app/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,18 @@
+.irs-reproofing-notice {
+  background-color: #007187;
+  color: #ffffff;
+}
+
+.irs-reproofing-notice:before {
+  background-image: url('alert/irs-info.svg') no-repeat center/contain;
```

Comment:
Don't we need an asset helper here to add the thumbprint in deployed environments?

Reply:
> Don't we need an asset helper here to add the thumbprint in deployed environments?

This should be [taken care of automatically by Sprockets](https://github.com/rails/sprockets-rails#:~:text=config.assets.resolve_assets_in_css_urls).

Reply:
In light of @aduth's comment, leaving as is. One this is in acceptance, we can double check that the asset shows up.